variables.tf

```diff
@@ -318,6 +318,7 @@ variable "addons" {
     cluster-autoscaler        = optional(string)
     vpc-block-csi-driver      = optional(string)
     ibm-storage-operator      = optional(string)
+    openshift-ai              = optional(string)
```

Comment:
We can omit including `openshift-ai` here. We will use addon DA to include this.